ci-operator/step-registry/ipi/conf/aws/custom-az/ipi-conf-aws-custom-az-commands.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
```

Comment:
suggest to add the following options like other scripts to be more restrict:
```
set -o nounset
set -o errexit
set -o pipefail
```

Reply:
fixed